src/js/components/Meter/doc.js

```diff
@@ -70,6 +70,14 @@ export const doc = Meter => {
 };
 
 export const themeDoc = {
+  'global.colors': {
+    description: 'color options',
+    type: 'object',
+    defaultValue: `{
+      "accent-1": "#6FFFB0",
+      "graph-0": "accent-1",
```

Comment:
why `"graph-1": "neutral-1",` isn't included in this case?

Reply:
The list can be rather long. I wanted to include at least a couple to give an example. Perhaps add a `...`?

Reply:
yes, the reason I brought it up was that it wasn't consistent with other places that showed the default value. `...` should work.

Reply:
done__